src/components/expanded-state/asset/ChartExpandedState.js

```diff
@@ -265,11 +264,10 @@ export default function ChartExpandedState({ asset }) {
     ),
   });
 
-  const uniswapAssetsInWallet = useUniswapAssetsInWallet();
+  const { sortedAssets } = useSelector(sortAssetsByNativeAmountSelector);
```

Comment:
redux/data - `accountAssetsData` - it's a map which takes in the token as the key